Tasks/Kozlovskii/Task2/task2_1.py

```diff
@@ -0,0 +1,79 @@
+hh = input("Enter time value (data in the format hh:mm):")
+a = hh.split(':')
+#print(a, type(a))
+b = [int(i) for i in a]
+#print(b, type(b))
+
+d = {}
+for key in ["07:00", "19:00"]:
+    d[key] = "семь часов ровно"
+for key in ["01:00", "13:00"]:
+    d[key] = "один час ровно"
+for key in ["02:00", "14:00"]:
+    d[key] = "два часа ровно"
+for key in ["03:00", "15:00"]:
+    d[key] = "три часа ровно"
+for key in ["04:00", "16:00"]:
+    d[key] = "четыре часа ровно"
+for key in ["05:00", "17:00"]:
+    d[key] = "пять часов ровно"
+for key in ["06:00", "18:00"]:
+    d[key] = "шесть часов ровно"
+for key in ["08:00", "20:00"]:
+    d[key] = "восемь часов ровно"
+for key in ["09:00", "21:00"]:
+    d[key] = "восемь часов ровно"
+for key in ["10:00", "22:00"]:
+    d[key] = "десять часов ровно"
+for key in ["11:00", "23:00"]:
+    d[key] = "одиннадцать часов ровно"
+for key in ["12:00", "24:00"]:
+    d[key] = "двенадцать часов ровно"
+res = d.get(hh)
```

Comment:
it's a very unorthodox way of creating a dictionary, would be much easier to just use some literals